app/src/main/java/io/seqera/wavelit/App.java

```diff
@@ -406,7 +402,21 @@ protected BuildContext prepareContext()  {
             return null;
         BuildContext result = null;
         try {
-            result = BuildContext.of(new Packer().layer(Path.of(contextDir)));
+            //check for .dockerignore file in context directory
+            Path dockerIgnorePath = Paths.get(contextDir, ".dockerignore");
+            if (Files.exists(dockerIgnorePath)){
+                List<String> dockerIgnorePatterns = new ArrayList<>();
+                dockerIgnorePatterns = Files.readAllLines(dockerIgnorePath);
+                Set<String> ignoredPatterns = new LinkedHashSet<>();
```

Comment:
I've realised now the need for `LinkedHashSet` is required because the order matter. However, being so, it would be better to use a List because pattern repetition may be allowed.

Reply:
I'll change it